Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q \
+	  doc/api doc/api_assets doc/guides src lib benchmark tools/doc/ tools/icu/
```

Comment:
I guess that whatever generates the changelogs should have escaped some symbols like <code>`_*~\\</code>, prefixing those with <code>\\</code>. But for the cases that are already there and that require manual fixing, perhaps using inline code blocks makes more sense where appropriate.